client-src/elements/datatypes.ts

```diff
@@ -1,6 +1,9 @@
+import {TemplateResult} from 'lit';
```

Comment:
```suggestion
import type {TemplateResult} from 'lit';
```

You should be able to `import type` since it is only being used as a type here.